tasks/76_deep-object-clone/deep-object-clone.js

```diff
@@ -0,0 +1,10 @@
+export const deepClone = ( obj ) => {
+  let result = {};
+  Object.keys( obj ).forEach(( key ) => {
+    const value = obj[key];
```

Comment:
you don't need additional function here, as you can call deepClone recessively directly, also you need call this function only for object and in other cases you just assign a value here you have an error like this { a: 1, b: { c:10 } } -> { a: {}, b: { c: {} }} but as result have to be copy of the same object

yep there is another task with bad tests :(

Reply:
Done